### DISCUSSION TITLE
I created a .NET 8 template using Linq2Db (also FluentMigrator and FastEndpoints)

### DISCUSSION THREAD

**Discussion by timbze** [linq2db/linq2db#4425] [Show and tell]
Maybe someone is interested. New .NET template with just basics.
https://github.com/timbze/template_linq2db

**Comment by viceroypenguin** [comment]
I love comparing templates. I look forward to reading this. If you want another one for comparison, check out mine: https://github.com/viceroypenguin/VsaTemplate
